browser/views.py

```diff
@@ -220,24 +221,43 @@ def _handle_known_person_form(request, extra_form, person):
 
 
 @staff_member_required
-def edit_person(request, person_id):
-    person = get_object_or_404(Person, pk=person_id)
+def edit_person(request, person_id=None):
```

Comment:
Won't the code be more readable if you just had a different view function for "add-person"?

Reply:
Ignore the previous comment. It seems like they are very similar in functionality.